src/sentry/api/bases/organization_events.py

```diff
@@ -269,6 +272,27 @@ def get_event_stats_data(
                     "tpm()": "tpm(%d)" % rollup,
                     "tps()": "tps(%d)" % rollup,
                 }
+                # For the new apdex, we need to add project threshold config as a selected
+                # column which means the group by for the time series won't work.
+                # As a temporary solution, we will calculate the mean of all the project
+                # level thresholds in the request and use the legacy apdex calculation.
```

Comment:
Can we add a `TODO(snql)` in the comments here so its clear that this can be fixed once we migrate this over?